core/controllers/galleries.py

```diff
@@ -123,29 +167,23 @@ class GalleryHandler(base.BaseHandler):
 
     def get(self):
         """Handles GET requests."""
-        # TODO(sll): Figure out what to do about explorations in categories
-        # other than those explicitly listed.
-
         query_string = self.request.get('q')
```

Comment:
Actually -- if there are double quotation marks in the query string (say), how does the search backend deal with it? Does it choke?

If so ... should we strip weird punctuation from the query string before feeding it to the backend?